client/packages/host/package.json

```diff
@@ -25,6 +25,7 @@
     "start-local": "webpack-cli serve --env API_HOST='http://localhost:8000'",
     "build": "yarn clean && webpack --env production",
     "serve": "serve dist -p 3003",
+    "clean": "rimraf ./dist/*",
```

Comment:
offroad: my suggestion for #64 which I have been using to build. sorry, it snuck in and I've left it in as I find it helpful

Reply:
if this is done, next time it's built, all build files will show in git, see: https://github.com/openmsupply/open-msupply/pull/65#discussion_r886220779

Reply:
removed